source/clear-linux/get-started/virtual-machine-install/virtualbox.rst

```diff
@@ -21,7 +21,7 @@ The instruction assumes that you have:
    
```

Comment:
Please update title/first paragraph to use CL-ATTR substitution. Also, under "Create a virtual machine in VirtualBox", replace "Clear Linux" in text with CL sub.